src/datepicker-input/datepicker-input.component.ts

```diff
@@ -84,4 +88,13 @@ export class DatePickerInput {
 	@Input() invalidText: string;
 
 	@Input() skeleton = false;
+
+	onChange() {
+		this.valueChange.emit(this.dateInput.nativeElement.value);
+		if (this.dateInput && this.dateInput.nativeElement.value === "") {
+			this.invalid = false;
```

Comment:
I'm not sure about this ... do we have anything else that does this by default? And how does this behave with the forms module?